### PR TITLE
calendar style

### PR DESCRIPTION
**Pull request by Rubenrrb**
I created a style for the calendar, hope you like it.

But still we need to figure it out, how to keep the text inside the box, and also I need to ask how can different events display in columns and now they are overlapping each other. but in general I think looks good.

image

### CODE REVIEW

**Comment by blamb888** [comment]
Hellz yeah! Looks guuud~ Let's have a morning merge party lol.

File: app/views/calendars/_day.html.erb [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   <%= @calendar.weekdays.keys[days_of_week_counter].capitalize %>
   <% @day_events = find_day_events(@events, @day_of_year)%>
   <% @day_events.each do |event|%>
     <%= render "calendars/event", :event=> event%>
     <%= render 'calendars/event-info-modal', event: event %>
   <% end %>
-  <i class="fas fa-plus-square cursor-pointer" data-toggle="modal" data-target="#event-modal-<%= @day_of_year %>"></i>
-  <%= render 'events/modal', day_of_year: @day_of_year %>
+  <div class="d-flex">
+    <i class="far fa-calendar-plus" data-toggle="modal" data-target="#event-modal-<%= @day_of_year %>"></i>
```
Comment (blamb888) [inline app/views/calendars/_day.html.erb:10]: `<i class="far fa-calendar-plus cursor-pointer" `?

Comment (blamb888) [inline app/views/calendars/_day.html.erb]: <i class="far fa-calendar-plus cursor-pointer" ?

Comment (Rubenrrb) [reply app/views/calendars/_day.html.erb]: yep u are right

**Review by blamb888 — APPROVED** [review]
Awesome! Thanks!

**Pull request merged**